src/app/app.service.ts

```diff
@@ -20,6 +20,7 @@ export class AppService {
   public allUsers: any[] = [];
   public isLoading: boolean = false;
   public materials = MATERIALS;
+  public title = 'INDIA WASTE EXCHANGE';
```

Comment:
why is this added in `app.service.ts`? it should have been added in `app.component.ts` and accessed in `app.component.html` as `{{ this.title }}`.

Reply:
I think component as controller and should not do or have anything other than mediator between view and models via services.

I am not sure how you are following here. So, I will make change here